telemetry/specs/INT.mdk

```diff
@@ -1036,20 +1039,28 @@ each bit corresponds to a specific standard metadata as specified in Section 3.
   inserted at each hop, except for bit 6. If bit 6 is set, the instruction requires 8 bytes of 
   metadata. Per-hop metadata length is set accordingly at the INT source.
   
-  * Domain Specific ID (16b): the unique ID of the INT Domain.
+* Domain Specific ID (16b): the unique ID of the INT Domain.
   
-  * DS Instruction (16b): Instruction bit map specific to the INT domain identified by the 
+* DS Instruction (16b): Instruction bit map specific to the INT domain identified by the 
   Domain Specific ID. Domain Specific Instruction is an instruction that requires additional 
   processing of Domain Specific Flags (DS Flags) for the INT Domain identified by Domain Specific ID. 
+
+  Some instruction bits can be defined as 'source-only' metadata by the INT domain. Those metadata
+  will be inserted only by the INT source, not by INT transit or sink devices.
```

Comment:
This is great.  Thanks for adding it!

Reply:
Thanks for your timely inputs from your usecase!